open_humans/templatetags/utilities.py

```diff
@@ -368,16 +368,29 @@ def template_bool(item):
 
 
 @register.simple_tag(takes_context=True)
-def get_next_url(context):
-    next = context['request'].GET.get('next', None)
-    if next:
-        return quote_plus(next)
+def get_next_url(context, include_next=False, social=False):
+    """
+    Get the url to assign to ?next=.
+    """
+    next_t = context['request'].GET.get('next', None)
+
+    if next_t:
+        param = quote_plus(next_t)
+    elif 'redirect_field_value' in context:
+        param = context['redirect_field_value']
```

Comment:
Should this have a `quote_plus` as well?

Maybe some more documentation in comments regarding how `quote_plus` should be used within redirects is good (e.g. "everything is always wrapped with this, and always unwrapped later" if that's the case)